src/index.js

```diff
@@ -0,0 +1,33 @@
+import './style.css';
+
+const grab = (e, isId = false, qAll = false) => {
+  if (isId) {
+    return document.getElementById(e);
+  } if (qAll) {
+    return document.querySelectorAll(`.${e}`);
+  }
+  return document.querySelector(`.${e}`);
+};
+
+const todo = [
+  {
+    description: 'Wash the dishes',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'complete To Do list project',
+    completed: false,
+    index: 1,
+  },
+];
+
```

Comment:
- [ ] Great job so far 👏🏻 , but according to your implementations, the first book in your task board (todo array) has the `completed` property set to `true` and the second book has the `completed` property set to `false` however, as you can see in the screenshot below 👇🏻 both books have the "completed" property set to true by default, please fix that 👍🏻

![todo](https://user-images.githubusercontent.com/79058364/177937299-9d54f2d4-c5d9-4196-8653-321f3f9033b1.png)